src/Build/Resources/Strings.resx

```diff
@@ -53,214 +112,214 @@
+  <data name="ConditionNotBooleanDetail" xml:space="preserve">
     <value>MSB4113: Specified condition "{0}" evaluates to "{1}" instead of a boolean.</value>
     <comment>{StrBegin="MSB4113: "}</comment>
   </data>
-  <data name="ConfigFileReadError" UESanitized="false" Visibility="Public">
+  <data name="ConfigFileReadError" xml:space="preserve">
     <value>MSB4136: Error reading the toolset information from the configuration file "{0}". {1}</value>
     <comment>{StrBegin="MSB4136: "}</comment>
   </data>
-  <data name="ConflictingValuesOfMSBuildToolsPath" UESanitized="false" Visibility="Public">
+  <data name="ConflictingValuesOfMSBuildToolsPath" xml:space="preserve">
     <value>MSB4142: MSBuildToolsPath is not the same as MSBuildBinPath for the ToolsVersion "{0}" defined at "{1}". If both are present they must have the same value.</value>
     <comment>{StrBegin="MSB4142: "}</comment>
   </data>
-  <data name="DefaultTasksFileFailure" UESanitized="false" Visibility="Public">
+  <data name="DefaultTasksFileFailure" xml:space="preserve">
     <value>MSB4009: The default tasks file could not be successfully loaded. {0}</value>
     <comment>{StrBegin="MSB4009: "}UE: This message is shown when one of the default tasks file (*.tasks) located alongside the MSBuild binaries cannot
     be opened/parsed. "{0}" contains a message explaining why. The filename itself is not part of the message but is provided
     separately to loggers.
     LOCALIZATION: "{0}" is a message from some FX method and is already localized.</comment>
   </data>
-  <data name="DefaultTasksFileLoadFailureWarning" UESanitized="false" Visibility="Public">
+  <data name="DefaultTasksFileLoadFailureWarning" xml:space="preserve">
     <value>MSB4010: The "{0}" files could not be successfully loaded from their expected location "{1}". Default tasks will not be available. {2}</value>
     <comment>{StrBegin="MSB4010: "}UE: This message is shown when the default tasks files that are located alongside the MSBuild binaries cannot be
     found, either because they don't exist, or because of lack of permissions. "{2}" contains a message explaining why.
     LOCALIZATION: "{2}" is a message from some FX method and is already localized.</comment>
   </data>
-  <data name="ImportIntroducesCircularity">
+  <data name="ImportIntroducesCircularity" xml:space="preserve">
     <value>Importing the file "{0}" into the file "{1}" results in a circular dependency.</value>
     <comment>
        {0} is a file imported into the file "{1}" such that it results in a circular dependency. For e.g. if t1.targets imports
        t2.targets and t2.targets tries to import t1.targets, then it results in a circular dependency.
     </comment>
   </data>
-  <data name="SearchPathsForMSBuildExtensionsPath">
-      <value>Search paths being used for {0} are {1}</value>
+  <data name="SearchPathsForMSBuildExtensionsPath" xml:space="preserve">
+    <value>Search paths being used for {0} are {1}</value>
```

Comment:
Did you edit this file with VS? Could I get you to revert it and go back and manually edit it? Lots of probably-not-needed changes in here.